assets/js/main.js

```diff
@@ -1,40 +1,64 @@
+/* eslint-disable max-classes-per-file */
+
+// Class constructor
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+class BooksCollection {
+  constructor() {
+    this.booksArray = [];
+  }
+
+  addBookToCollection = (title, author) => {
+    const newBook = new Book(title, author);
+    this.booksArray.push(newBook);
```

Comment:
[Optional]
You should use array methods that doesn't mutate the array, e.g. [`concat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat), [`filter`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter), [`slice`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) instead of [`push`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push), [`pop`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/pop), [`splice`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice).

Mutation hides change, which creates unexpected side effects, which can cause nasty bugs. When you enforce immutability you can keep your application architecture and mental model simple, which makes it easier to reason about your application